python/lsst/obs/pfs/constructNormFlatTask.py

```diff
@@ -137,7 +95,8 @@ def combine(self, cache, struct):
         for expRef in dataRefList:
             exposure = expRef.get('postISRCCD')
             xOffsets.append(exposure.getMetadata().get('sim.slit.xoffset'))
-            fts = myFindTask.run(exposure)
+            fiberTrace = expRef.get('fiberTrace')
+            fts = 
```

Comment:
This looks like a commit that can't even have been importable.  How did your tests not catch this?